app/models/candidate.rb

```diff
@@ -0,0 +1,5 @@
+class Candidate < ApplicationRecord
+  devise
```

Comment:
We talked about the unwelcome devise voodoo when we discussed our options. Is it worth a comment here explaining why we haven't got any Devise modules enabled?